js/okx.js

```diff
@@ -3566,36 +3632,11 @@ module.exports = class okx extends Exchange {
          * @param {object} params extra parameters specific to the okx api endpoint
          * @returns {object} an [address structure]{@link https://docs.ccxt.com/en/latest/manual.html#address-structure}
          */
-        const rawNetwork = this.safeStringUpper (params, 'network');
```

Comment:
all these lines below are similar as it was in huobi/bybit and just simply replaced by our base network methods